chain.go

```diff
@@ -26,10 +30,13 @@ func (c *Chain) GetBlockHash(blockNumber int) string {
 	return response.Result.(string)
 }
 
-func (c *Chain) GetBlockByNumber(blockNumber int) interface{} {
+func (c *Chain) GetBlockByNumber(blockNumber int) Block {
 	const method = "chain_getBlockByNumber"
 	response := c.rpcClient.call(callInterface{method: method, id: ""}, blockNumber)
```

Comment:
How about pass the block to the callInterface function?

Reply:
Do you mean `blockNumber`?

Reply:
Sorry I mean passing the `block` variable to `call` function as an argument.

Reply:
I see. I think that's a good idea.